src/api/app/views/webui2/webui/user/index.html.haml

```diff
@@ -4,3 +4,6 @@
 .card.mb-3
   = render partial: 'webui/configuration/tabs'
   .card-body
+    = link_to(user_register_user_path) do
+      %i.fas.fa-user-plus
```

Comment:
The CSS classes for the link should be `fas.fa-plus-circle.text-primary` to be consistent with the other views.